.github/workflows/stale-issue-gardening.yml

```diff
@@ -1,52 +1,53 @@
+                      days-before-close: -1
+                      only-labels: '[Status] Needs More Info'
+                      remove-stale-when-updated: true
+                      stale-issue-label: '[Status] Stale'
+                    - name: 'Issues that have needed testing for a period of time'
+                      message: "Hi,\nThis issue has gone 30 days without any activity. This means it is time for a check-in to make sure it is still relevant. If you are still experiencing this issue with the latest versions, you can help the project by responding to confirm the problem and by providing any updated reproduction steps.\nThanks for helping out."
+                      days-before-stale: 30
+                      days-before-close: -1
+                      only-labels: 'Needs Testing'
+                      remove-stale-when-updated: true
+                      stale-issue-label: '[Status] Stale'
+                    - name: 'Flaky test issues without activity'
+                      message: 'This issue has gone 30 days without any activity.'
+                      days-before-stale: 30
+                      days-before-close: 1
+                      only-labels: '[Type] Flaky Test'
+                      remove-stale-when-updated: true
+                      stale-issue-label: '[Status] Stale'
+                    - name: 'Issues without recent updates that need confirmation'
+                      message: "Hi,\nThis issue has gone 180 days without any activity. This means it is time for a check-in to make sure it is still relevant. If you are still experiencing this issue with the latest versions, you can help the project by responding to confirm the problem and by providing any updated reproduction steps.\nThanks for helping out."
+                      days-before-stale: 180
+                      days-before-close: -1
+                      remove-stale-when-updated: false
+                      stale-issue-label: 'Needs Testing'
 
-    steps:
-      - name: Update issues
-        uses: actions/stale@5ebf00ea0e4c1561e9b43a292ed34424fb1d4578 # v6.0.1
-        with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-          stale-issue-message: ${{ matrix.message }}
-          days-before-stale: ${{ matrix.days-before-stale }}
-          days-before-close: ${{ matrix.days-before-close }}
-          remove-stale-when-updated: ${{ matrix.remove-stale-when-updated }}
-          stale-issue-label: ${{ matrix.stale-issue-label }}
+        steps:
+            - name: Update issues
+              uses: actions/stale@5ebf00ea0e4c1561e9b43a292ed34424fb1d4578 # v6.0.1
+              with:
+                  repo-token: ${{ secrets.GITHUB_TOKEN }}
+                  stale-issue-message: ${{ matrix.message }}
+                  days-before-stale: ${{ matrix.days-before-stale }}
+                  days-before-close: ${{ matrix.days-before-close }}
+                  only-labels: ${{ matrix.only-labels }}
```

Comment:
From what I understand, an unset value is ok - https://github.com/actions/stale#only-labels:

> If unset (or an empty string), this option will not alter the stale workflow.